library/alloc/src/lib.rs

```diff
@@ -106,10 +106,10 @@
 #![feature(char_max_len)]
 #![feature(clone_to_uninit)]
 #![feature(coerce_unsized)]
+#![feature(const_convert)]
 #![feature(const_default)]
 #![feature(const_eval_select)]
 #![feature(const_heap)]
-#![feature(const_trait_impl)]
```

Comment:
Was listed under library features, and is actually a language feature. Since the diff doesn't make that clear.